src/util/server/formatMDFiles.ts

```diff
@@ -0,0 +1,27 @@
+import type { MDFiles } from 'custom-types';
+
+export const formatMDFiles = (mdFiles: MDFiles[]): MDFiles[] => {
+  const formattedMDFiles = mdFiles.map(({ header, files, directories }) => {
+    header = checkSortCharacters(header);
+    files = files.map(({ subHeader, html }) => {
+      subHeader = checkSortCharacters(subHeader);
+      return { subHeader, html };
+    });
+    directories = formatMDFiles(directories);
+
+    return { header, files, directories };
+  });
+
+  return formattedMDFiles;
+};
+
+export const checkSortCharacters = (value: string): string => {
+  if (value[0] === '_') {
+    value = value.substring(1);
+  }
+  if (!isNaN(parseInt(value[0]))) {
+    value = value.substring(3);
+  }
```

Comment:
I would do this as a regex instead of hardcoding position 3 because we can get into situations like this:

![Screen Shot 2021-09-20 at 5 01 52 PM](https://user-images.githubusercontent.com/31771087/134075271-e53c037b-9ad6-4ad0-9d64-28cd517f3450.png)


Other alternative is to have prebuild validation on the pattern of the dir naming

Reply:
Other alternative is to have prebuild validation on the pattern of the dir naming

Reply:
Yeah I was kinda wondering what the best approach would be to tackle this problem. You're solution is probably the best, just use a regex.

Reply:
What did you name the folder so I can make sure this doesnt happen after the changes.